apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -239,8 +267,8 @@ export class SendMessageChat extends SendMessageBase {
       })
     );
 
-    if (chatWebhookUrl && integration) {
-      await this.sendMessage(chatWebhookUrl, integration, content, message, command, channelSpecification);
+    if ((chatWebhookUrl && integration) || (phoneNumber && integration)) {
```

Comment:
In an attempt to start defining a naive strategy pattern until we do the proper refactoring, I want to suggest introducing two private methods:

- sendChannelMessageViaWebhookUrl
- sendChannelMessageViaAPIKey

That way, we will ensure that messages created in Mongo have only defined fields per use case and have fewer and shorter if conditions across the file. 

The current `sendError` can be implemented inside the above two private methods separately for now.